src/main/java/org/datadog/jenkins/plugins/datadog/listeners/DatadogGraphListener.java

```diff
@@ -104,6 +104,7 @@ public void onNewHead(FlowNode flowNode) {
             // Add custom result tag
             TagsUtil.addTagToTags(tags, "result", result);
             client.gauge("jenkins.job.stage_duration", getTime(startNode, endNode), hostname, tags);
+            client.gauge("jenkins.job.stage_completed", 1, hostname, tags);
```

Comment:
```suggestion
            client.incrementCounter("jenkins.job.stage_completed", hostname, tags);
```

Reply:
@FlorianVeaux - 
Thank you very much for your review and applied the suggested change and also updated the ReadMe.

Reply:
I tried updating the ReadMe but the jenkins build continues to error out with
Run DataDog/labeler@glob-all
Error: HttpError: Resource not accessible by integration
Error: Resource not accessible by integration

Let me know if there is anything additional that I can do here to help.